Bottom: b1f08e5c6b8ae127e05ad3273fb33233b411c00b
Top:    95460fe5374d44c039db936540a7cc68bbf1716c
Author: Edward "Teddy" Brow <edward.teddy.brow@gmail.com>
Date:   2017-04-14 19:52:39 -0700

exercise 1.12


---

diff --git a/01_12/main.cc b/01_12/main.cc
new file mode 100644
index 0000000..869cf97
--- /dev/null
+++ b/01_12/main.cc
@@ -0,0 +1,9 @@
+#include <iostream>
+
+int main()
+{
+  int sum = 0;
+  for (int i = -100; i <= 100; ++i)
+    sum += i;
+  std::cout << "sum: " << sum << std::endl;
+}
diff --git a/01_12/notes.txt b/01_12/notes.txt
new file mode 100644
index 0000000..e843d9a
--- /dev/null
+++ b/01_12/notes.txt
@@ -0,0 +1,2 @@
+The for loop sums all numbers in range [-100, 100]. The final value of sum is 0, since the contribution of each integer
+in the range is "cancelled out" by its negative.
